components/cd-tag/cd-tag.css

```diff
@@ -0,0 +1,75 @@
+/**
+ * Base component styles for the collection of tags
+ */
+.cd-tags {
+  /* Override container class if you wish to provide specific styling to all tags */
+}
+
+/**
+ * Individual tag
+ *
+ * - margin is set in `rem` to ensure that multiple tag sizes visually line up
+ * - all other units are `em` so that setting `font-size` can scale them easily
+ */
+.cd-tag {
+  display: inline-block;
+  margin: 0.25rem;
+  padding: 0 .25em;
```

Comment:
REMs please

Reply:
Per the comment block, converting this particular unit to `rem` could result in a disproportional-looking component when the different sizes are used. I set up the demo page to show this exact effect (or rather, how nice it can look with many different-sized tags)

But if `em` truly has to go, then I have another idea to further componentize this.